pom.xml

```diff
@@ -415,10 +437,20 @@
             <artifactId>jackson2-geojson</artifactId>
             <version>0.8</version>
         </dependency>
+        <dependency>
+            <groupId>org.mapdb</groupId>
+            <artifactId>elsa</artifactId>

```

Comment:
I don't see elsa being used anywhere in this pull request. I believe I was experimenting with it sometime in the past. Is it possible that this PR is reverting some changes we have made since your branch diverged?